todo_app/templates/index.html

```diff
@@ -11,8 +11,16 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li>
+            {{ item.title }}
+          </li>
+        {% endfor %}
       </ul>
+      <form method="post" action="/new">
+        <input name="item" type="text"/>
```

Comment:
Might be nice to make this required just to improve the client-side UX a little
```suggestion
        <input name="item" type="text" required/>
```